src/improvements/justfile

```diff
@@ -74,3 +74,12 @@ monorepo-integration-test:
     export SUPERCHAIN_OPS_ALLOCS_PATH=./allocs.json
     cd ${root_dir}/lib/optimism/packages/contracts-bedrock/ && just test-upgrade
     rm -f ${allocs_path} # clean up
+
+check-superchain-registry-latest:
+    #!/usr/bin/env bash
+    set -euo pipefail
+    root_dir=$(git rev-parse --show-toplevel)
+    bash ${root_dir}/src/improvements/script/check-superchain-latest.sh
+    (cd ${root_dir}/lib/superchain-registry && git pull origin main)
+    bash ${root_dir}/src/improvements/script/check-superchain-latest.sh
+    rm -rf ${root_dir}/hash.txt
```

Comment:
Why do we do the check this way? Can't we just `sha256sum` against the contents of 
- `/lib/superchain-registry/superchain/extra/addresses/addresses.json` vs.
- https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/extra/addresses/addresses.json

And not deal with temp files, git pull, and running this script twice

Reply:
good point: https://github.com/ethereum-optimism/superchain-ops/pull/574/commits/0a4e5121dda3bd02f22c4d8f5b62b588d38e7570